src/lib.rs

```diff
@@ -572,13 +572,32 @@ impl TestKit {
         self.poll_events();
     }
 
+    /// Creates block with the given transactions.
+    /// Transactions that are in mempool will be ignored.
+    pub fn create_block_with_transactions(&mut self, txs: Vec<Box<Transaction>>) {
+        let tx_hashes = {
+            let mut mempool = self.mempool.write().expect(
+                "Cannot write transactions to mempool",
+            );
+
+            let mut tx_hashes = Vec::new();
```

Comment:
Nitpick: maybe, use `Vec::with_capacity(txs.len())` here? We know the size of the `tx_hashes` vector beforehand.

Reply:
Yes